libxrdp/libxrdp.c

```diff
@@ -1332,13 +1332,13 @@ libxrdp_fastpath_send_surface(struct xrdp_session *session,
     s->rdp_hdr = s->sec_hdr + sec_bytes;
     s->end = data_pad + pad_bytes + data_bytes;
     s->p = s->data + (rdp_bytes + sec_bytes);
-    /* TS_SURFCMD_SET_SURF_BITS */
-    out_uint16_le(s, 0x0001); /* CMDTYPE_SET_SURFACE_BITS */
+    /* TS_SURFCMD_STREAM_SURF_BITS */
+    out_uint16_le(s, 0x0006); /* CMDTYPE_STREAM_SURFACE_BITS */
```

Comment:
Why not just use the CMDTYPE_STREAM_SURFACE_BITS? It's defined and it's even used in xrdp_surface.c